benchmarks/functions/__init__.py

```diff
@@ -45,6 +45,24 @@ def rec(x):
         return None if x is None else tuple(rec(x))
 
 
+    def _check_format(self, outputs, grad_outputs):
+        """Checks if both of the inputs have the same format.
+        """
+        outputs_is_list = isinstance(outputs, (list, tuple))
+        grad_outputs_is_list = isinstance(grad_outputs, (list, tuple))
+        if (outputs_is_list and grad_outputs_is_list):
+            if len(outputs) != len(grad_outputs):
+                msg = ("Number of outputs and grad_outputs mismatch.\n" +
+                       "outputs : %d != grad_outputs : %d\n" %
```

Comment:
* I think printing length is confusing as this method may applied recursively.
* To make it more understandable to users (who writes benchmark code), how about saying `Format of grad_outputs is not in the same format as the output of target function`?

Reply:
Fixed.

Reply:
Fixed.